okhttp/src/main/kotlin/okhttp3/FormBody.kt

```diff
@@ -92,12 +92,12 @@ class FormBody internal constructor(
     fun add(name: String, value: String) = apply {
       names += name.canonicalize(
           encodeSet = FORM_ENCODE_SET,
-          plusIsSpace = true,
+          plusIsSpace = false,
```

Comment:
This probable warrants an inline comment because the parameter name diverts from the intention. 

> @param plusIsSpace true to encode '+' as "%2B" if it is not already encoded.

But plusIsSpace, we just don't want the additional encoding.